docs/detections/rules-ui-manage.asciidoc

```diff
@@ -120,6 +120,11 @@ If you try to bulk edit with both prebuilt and custom rules selected, the action
 * Bulk edit multiple rules: Select the rules you want to edit, then select an action from the *Bulk actions* menu:
 ** *Index patterns*: Add or delete the index patterns used by all selected rules.
 ** *Tags*: Add or delete tags on all selected rules.
+** *Add rule actions*: Add <<rule-notifications,notification actions>> on all selected rules.
```

Comment:
Should we note that `Add rule actions` can be applied to prebuilt rules as well?

Looking at this note above the list of bulk edit, it might be a bit confusing, in which cases action will affect prebuilt rules, in which not.

<img width="924" alt="Screenshot 2022-10-13 at 16 03 44" src="https://user-images.githubusercontent.com/92328789/195634229-2efa7026-01b0-459f-b87a-b1d1b24c1a6c.png">

> If you try to bulk edit with both prebuilt and custom rules selected, the action will affect only custom rules.

Also, it looks like this note doesn't reflect[ appearance of modal window,](https://github.com/elastic/kibana/issues/124918#issuecomment-1199146225), which happens when user selects both custom and prebuilt rules. And can decide which action to take: edit only custom rules OR cancel action at all

Reply:
Thanks @vitaliidm! I agree, that note isn't quite accurate, so I revised it:

> If you try to bulk edit with both prebuilt and custom rules selected, the action will affect only the rules that can be modified.

The modal's UI text already does a good job of explaining the situation and what options the user has, so I don't think it's necessary to comment any further in docs; there isn't much more we could say, so we'd probably just be repeating what the UI said.